Makefile.am

```diff
@@ -88,9 +88,9 @@ else
    TOOLS_DIR=
 endif
 
-SUBDIRS = src $(TESTSERIAL_DIR) $(TESTPARALLEL_DIR) bin utils $(TOOLS_DIR) . \
+SUBDIRS = src $(TESTSERIAL_DIR) $(TESTPARALLEL_DIR) bin $(TOOLS_DIR) utils . \
```

Comment:
We need to make sure but I think utils is an optional component.

Reply:
I'm only changing the order of when the utils directory gets traversed..  Am I missing something in whether the order is important, i.e. I wanted to make sure that utils got visited AFTER the tools.   I should probably add a conditional statement to prevent dh5walk from be built if we don't build tools...

Reply:
> 
> 
> I'm only changing the order of when the utils directory gets traversed.. Am I missing something in whether the order is important, i.e. I wanted to make sure that utils got visited AFTER the tools. I should probably add a conditional statement to prevent dh5walk from be built if we don't build tools...

Assuming mirror_vfd should always be built, with or without tools, as long as the conditional only applies to dh5walk, I suppose if tools aren't built, then utils/mirror_vfd will be built next after bin.

Reply:
I'm thinking utils is an optional component therefore you would need to use a variable -> $(UTILS_DIR)
And anyway, if tools is not built, can anything in utils be built? Still I think utils is optional.

Reply:
I added a conditional check the utils/Makefile.am
if BUILD_TOOLS_CONDITIONAL
   TOOLS_DIR =tools
else
   TOOLS_DIR=
endif